tests/foreman/cli/test_hostcollection.py

```diff
@@ -157,7 +157,7 @@ def test_positive_update_to_unlimited_hosts(module_org):
         {
             'name': host_collection['name'],
             'organization-id': module_org.id,
-            'unlimited-hosts': True,
+            'unlimited-hosts': "True",
```

Comment:
What changed that made this pass in a string now instead of a boolean?  And please check your PR run, it failed.

Reply:
@latran `--unlimited-hosts` was changed from an option to a flag according to this PR: https://github.com/Katello/hammer-cli-katello/pull/846 
because of this issue: https://projects.theforeman.org/issues/34889
This was introduced in Katello 4.5, so this should only affect 6.12+


`Error: Option '--unlimited-hosts': Value must be one of true/false, yes/no, 1/0`

When passing a boolean directly here, `_construct_command()` in `cli/base.py` appends the key as an option `--unlimited-hosts` rather than the value itself to make it a flag `--unlimited-hosts=True` and also adds quotes:

```
for key, val in options.items():
    if val is None:
        continue
    if val is True:
        tail += f' --{key}'
    elif val is not False:
        if isinstance(val, list):
            val = ','.join(str(el) for el in val)
        tail += f' --{key}="{val}"'
cmd = f"{cls.command_base} {cls.command_sub or ''} {tail.strip()} {cls.command_end or ''}"
```